src/index.js

```diff
@@ -1,39 +1,124 @@
+/* eslint no-alert: "error" */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
```

Comment:
- [ ] Please remove the disabling linters, remember they're there to help you write better code. Writing a good and understandable code is very important

Reply:
- [ ] Greate work re-enabling the linters 👏🏾